rpm/openconnect.spec

```diff
@@ -69,6 +65,8 @@ mkdir -p %{buildroot}%{_docdir}/%{name}-%{version}
 %postun -p /sbin/ldconfig
 
 %files -f %{name}.lang
+%exclude %{_libexecdir}/openconnect/tncc-emulate.py
```

Comment:
I would add comment here that these depend on python which we want to avoid in our image.

Reply:
Will add ;-)

Reply:
done